pkg/api/handlers.go

```diff
@@ -39,6 +43,35 @@ func newHandler(context *clusterd.Context, config *Config) *Handler {
 	}
 }
 
+func (h *Handler) Init() error {
```

Comment:
You are right. Only the lowercase `init` function has the special role. But i would still rename it just to avoid any confusion.